README.md

```diff
@@ -251,23 +278,19 @@ of us having to split out the namespace...
 
 ## Database access
 
-...is controlled by [`lib/ht_traject/ht_dbh.rb`]
-(lib/ht_traject/ht_dbh.rb) with passwords/etc. in `lib/ht_secure_data.rb` 
-(not in this repo, obviously). 
-
-TODO: Put connection/ authentication info in ENV (and/or eventually k8s 
-secrets)
+Controlled by [`lib/ht_traject/ht_dbh.rb`](lib/ht_traject/ht_dbh.rb)
+with default passwords/etc. in `docker-compose` and `.env`, and -- eventually -- k8s secrets.
 
 ## Environment variables
 
   * `SOLR_URL` with the solr _core_ URL (i.e, ending in `/catalog`)
   * `REDIRECT_FILE` (optional) if you don't want to use the default 
     redirect file
-  * `NO_DB` if you want to skip all the database stuff. Useful for testing.
+  * ~~`NO_DB` if you want to skip all the database stuff. Useful for testing.~~
```

Comment:
I ignore the flag in the tests and code I've been working on. I haven't removed the flag from any of the existing code because IIRC I hadn't yet checked where else it might be used. I guess that's a question for @billdueber: does it make sense to preserve and honor the `NO_DB` flag?